Extension/src/main.ts

```diff
@@ -306,7 +306,7 @@ function makeOfflineBinariesExecutable(installBlob: InstallBlob): Thenable<void>
     packages.forEach(p => {
         if (p.binaries && p.binaries.length > 0 &&
             p.platforms.findIndex(plat => plat === installBlob.info.platform) !== -1 &&
-            p.architectures.findIndex(arch => arch === installBlob.info.architecture) !== - 1) {
+            (p.architectures == undefined || p.architectures.findIndex(arch => arch === installBlob.info.architecture) !== - 1)) {
```

Comment:
always use `===` over `==`.  Is there a linting rule for this?

Reply:
Yep. ```"triple-equals": true```

Reply:
Ok.

Reply:
So should I add triple-equal to tslint and fix all the non-triple equals (a lot)?

Reply:
I'll do that....

Reply:
Talked with @sean-mcmanus, delaying the triple-equal tslint to another PR.

Reply:
I just changed this 1 instance to ===.